### PR TITLE
Add end to end tests

### PR DESCRIPTION
**Pull request by octogradiste**
Adds end-to-end tests for both of our user flows.

### CODE REVIEW

**Comment by octogradiste** [comment]
We still need to integrate the user profile creation page in our end-to-end tests. Waiting for #188 to be merged.

**Comment by octogradiste** [comment]
Note that this will close #150.

**Review by ntissieres — DISMISSED** [review]
The user flows are perfectly executed and tested, thank you for the nice work !

File: app/src/androidTest/java/com/github/swent/echo/MainActivityTest.kt [hunk 1]
```diff
@@ -79,4 +106,147 @@ class MainActivityTest {
...
+
+        // Fill the event creation form
+        val title = "This is a test event title"
+        val description = "This is a test event description"
+        composeTestRule.onNodeWithTag("Title-field").performTextInput(title)
+        composeTestRule.onNodeWithTag("Description-field").performTextInput(description)
+        composeTestRule.onNodeWithTag("nb-participant-field").performTextReplacement("10")
+
+        // Click outside of the text field
+        composeTestRule.onNodeWithTag("Title-field").requestFocus()
```
Comment (violoncelloCH) [inline app/src/androidTest/java/com/github/swent/echo/MainActivityTest.kt:203]: why is this performed here ? does event creation fail otherwise ?

**Review by violoncelloCH — APPROVED** [review]
Nice! Only thing that comes to mind is that we could actually test setting more / all fields in the event creation/edit. But that can go to a new task I think :)

**Pull request merged**